tests/lightning/io/test_api.py

```diff
@@ -64,3 +68,10 @@ def test_reload_ckpt(self, tmpdir, partial_function_with_pos_and_key_args):
 
         loaded_func = loaded.extra["dummy"]
         assert loaded_func(b=2) == partial_function_with_pos_and_key_args(b=2)
+
+        model_yaml = Path(tmpdir) / "model.yaml"
+        assert model_yaml.exists()
+
+        observed = yaml.safe_load(model_yaml.read_text())
+        expected = yaml.safe_load((Path(ARTIFACTS_DIR) / "model.yaml").read_text())
```

Comment:
Is this the same as the old `model_config.yaml` from Nemo 1.0?

Reply:
No the format will be a bit different, an example artifact is included in the PR